src/System.Net.Requests/src/System/Net/HttpWebRequest.cs

```diff
@@ -1450,7 +1442,14 @@ private DateTime GetDateHeaderHelper(string headerName)
                 {
                     return DateTime.MinValue; // MinValue means header is not present
                 }
-                return HttpDateParse.StringToDate(headerValue);
```

Comment:
I don't see this StringToDate method being deleted in this PR.  If it's still being used, why expand it out, and if it's not being used, it should be deleted.

Reply:
Hrm, it probably got reintroduced when I rebased.